components/EditCollectiveForm.js

```diff
@@ -621,6 +621,10 @@ class EditCollectiveForm extends React.Component {
               text-align: center;
               margin: 1rem 0 3rem 0;
             }
+            #learnMore {
+              float: right;
+              padding-top: 8px;
+            }
```

Comment:
We should avoid styling through this method, especially using an `id`. Another way to achieve this may be to set `display="flex" justifyContent="space-between" flexWrap="wrap"` on the `Container`